syncd/ConcurrentQueue.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
```

Comment:
Since you are adding only queue here you should change this pr to add concurentqueue etc